src/tools/elektrad/doc_handler.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"net/http"
+)
+
+// getDocHandler redirects to the API documentation.
+//
+// Response Code:
+//		301 Moved Permanently Redirect to the documentation.
+//
+// Example: `curl localhost:33333`
+func (s *server) getDocHandler(w http.ResponseWriter, r *http.Request) {
+	http.Redirect(w, r, "https://elektrad.docs.apiary.io", 301)
```

Comment:
Isn't the info there outdated?

Reply:
well the functionality stayed the samed so i think it should still apply?

Reply:
You know best if it still applies : wink: What about linking to https://master.libelektra.org/doc/api_blueprints/elektrad.apib instead?